lib/node_modules/@stdlib/math/base/special/factorial2/lib/factorial2.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+// MODULES //
+
+var exp2 = require( '@stdlib/math/base/special/exp2' );
+var gamma = require( '@stdlib/math/base/special/gamma' );
+var isnan = require( '@stdlib/math/base/utils/is-nan' );
+var isInteger = require( '@stdlib/math/base/utils/is-integer' );
+var isNegativeZero = require( '@stdlib/math/base/utils/is-negative-zero' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var DOUBLE_FACTORIALS = require( './doublefactorials.json' );
+
+
+// VARIABLES //
+
+var MAX_FACTORIAL = 170; // TODO: consider extracting as a constant
+var PIO2 = 1.57079632679489661923; // pi/2
```

Comment:
Indeed. We have an internal backlog of various clean-up tasks. :( Currently picking them off as we have time. Thanks for the reminder!